astronomer/providers/apache/livy/triggers/livy.py

```diff
@@ -0,0 +1,133 @@
+"""This module contains the Apache Livy Trigger."""
+import asyncio
+from typing import Any, AsyncIterator, Dict, Optional, Tuple, Union
+
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+from astronomer.providers.apache.livy.hooks.livy import BatchState, LivyHookAsync
+
+
+class LivyTrigger(BaseTrigger):
+    """
+    Check for the state of a previously submitted job with batch_id
+
+    :param batch_id: Batch job id
+    :param spark_params: Spark parameters
+    :param livy_conn_id: reference to a pre-defined Livy Connection.
+    :param polling_interval: time in seconds between polling for job completion. Don't poll for values >=0
```

Comment:
This too

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/8fab5a5be069fd3a8e67ff4fe79d125577df04dd)